### PR TITLE
[CMS-242] Homepage Navbar + Responsive Hamburger Menu

### PR DESCRIPTION
**Pull request by james-teng-0**
- created a very basic navbar
- haven't implemented responsiveness yet - but i've added media queries for the hamburger icon.

image

### CODE REVIEW

**Review by fafnirZ — CHANGES_REQUESTED** [review]
please export the hamburger icon as an svg instead of a png, that way we can use code to edit the icon instead of requiring to import another different coloured svg

**Comment by kylee-software** [comment]
### Changes:

- Added SVGs for hamburger menu icon and  close-menu icon
- Made hamburger menu responsive 

**Navbar for Desktop:**
desktop view


**Hamburger Menu Icon:**
menu_close


**Hamburger Menu for Tablet:**
tablet_menu


**Hamburger Menu for Mobile:**
mobile_menu

File: Next/src/components/navbar/HamburgerMenu-styled.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import styled from "styled-components";
+
```
Comment (fafnirZ) [inline Next/src/components/navbar/HamburgerMenu-styled.tsx:2]: add comment somewhere to document the specific breakpoint sizes

File: Next/src/components/navbar/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import HamburgerIcon from "../../../public/assets/menu_icon.svg";
+import Image from 'next/image';
+import { NavbarOpenProps } from "./types";
+
+import {
+    Container, ItemWrapper,
+    NavItem, HamburgerButton
+} from "./Navbar-styled";
```
Comment (fafnirZ) [inline Next/src/components/navbar/Navbar.tsx:9]: strange import, when breaking imports into multi lines, usually only 1 per line

**Review by fafnirZ — APPROVED** [review]
lgtm, couple of minor changes

**Comment by fafnirZ** [comment]
@james-teng-0 can you take over, fix the 3 line changes and merge pls

**Pull request merged**